kqueen/engines/openstack_kubespray.py

```diff
@@ -807,7 +813,40 @@ def _get_userdata(self):
         }
         return "#cloud-config\n" + yaml.dump(userdata)
 
-    def _boot_servers(self, *, name, servers_range, image, flavor, network,
+    def _set_up_security_groups(self):
+        master_sg = self.c.get_security_group(MASTER_SECURITY_GR)
```

Comment:
imo all getters should be in one place in `init` method like any previous engines

Reply:
in init method data, collected from user is evaluated